### ISSUE TITLE
Inconsistent time format in API output

### ISSUE THREAD

**Issue by tbnguyen1407** [aler9/rtsp-simple-server#1093]
## Which version are you using?

v0.20.0

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [x] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

Inconsistent time formats used by "created" and "lastRequest" of hlsMuxer

```
GET {{baseUrl}}/v1/hlsmuxers/list

{
  "items": {
    "path1": {
      "created": "2022-08-17T15:16:08.122224+08:00",
      "lastRequest": "2022-08-17 15:16:39 +0800 +08"
    }
  }
}
```

## Describe how to replicate the issue

1. start the server
2. publish with any valid url
3. read HLS output with ffplay
4. call API to list hlsMuxers

## Did you attach the server logs?

no

## Did you attach a network dump?

no

## Additional notes

Would be better if there is an option for raw epoch instead of formatted string for time fields.

**Comment by aler9** [comment]
Thanks, now timestamps are always encoded with RFC3339.

**Comment by aler9** [comment]
fixed in v0.20.1
